bugsnag/django/__init__.py

```diff
@@ -69,6 +70,8 @@ def configure():
     middleware = bugsnag.configure().internal_middleware
     middleware.before_notify(add_django_request_to_notification)
 
+    bugsnag.configure().runtime_versions['django'] = django.__version__
```

Comment:
Should there be something in tests/integrations/test_django_1.py that checks for the presence of this entry? (similar question applies to the Flask integration too)